src/web/lib/hakuneko/frontend@classic-dark/menu.html

```diff
@@ -142,6 +147,13 @@
                 <div class="logo">
                     <img src="./img/logo_m.png"/>
                 </div>
+                <div class="contact">
```

Comment:
There is already a discord link in the `Help & Info` section which could be removed to avoid redundancy